jena/src/main/java/org/apache/commons/rdf/jena/JenaRDFTerm.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.commons.rdf.jena;
+
+import org.apache.commons.rdf.api.RDFTerm;
+import org.apache.jena.graph.Node;
+
+/** Access the Jena node backing this object */
+public interface JenaRDFTerm extends RDFTerm {
```

Comment:
Did you think about making this generic i.e.
```
public interface JenaRDFTerm<NodeType extends Node> extends RDFTerm {
        public NodeType asJenaNode();
```
and
```
class AbstractJenaRDFTerm<NodeType extends Node> implements JenaRDFTerm<NodeType>, RDFTerm {
+	private NodeType node;
```
and then for example:
```
public interface JenaIRI extends JenaRDFTerm<Node_URI>, IRI {
```
etc. That's more verbose, but maybe a bit safer?